op-ufm/go.mod

```diff
@@ -24,8 +24,8 @@ require (
 	github.com/Microsoft/go-winio v0.6.1 // indirect
```

Comment:
Can you explain more about why there is some changes in our module?

Reply:
Maybe these changes are from my IDE, it indexed the go.mod automatically. Let me revert go.mod and go.sum files